mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -183,65 +183,62 @@ public static AsyncHttpClient getAsyncHttpClient() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         Log.d(TAG, "onCreate()");
-        // Check if we are in development mode
-        isDeveloper = false;
+
         // Set default values, false means do it one time during the very first launch
         PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
+
         // Set non-persistent HABDroid version preference to current version from application package
         try {
             Log.d(TAG, "App version = " + getPackageManager().getPackageInfo(getPackageName(), 0).versionName);
             PreferenceManager.getDefaultSharedPreferences(this).edit().putString(Constants.PREFERENCE_APPVERSION,
                     getPackageManager().getPackageInfo(getPackageName(), 0).versionName).commit();
         } catch (PackageManager.NameNotFoundException e1) {
-            if (e1 != null)
-                Log.d(TAG, e1.getMessage());
+            Log.d(TAG, e1.getMessage());
         }
+
         checkDiscoveryPermissions();
         checkVoiceRecognition();
+
         // initialize loopj async http client
         mAsyncHttpClient = new MyAsyncHttpClient(this);
+
         // Set the theme to one from preferences
         mSettings = PreferenceManager.getDefaultSharedPreferences(this);
+
         // Disable screen timeout if set in preferences
         if (mSettings.getBoolean(Constants.PREFERENCE_SCREENTIMEROFF, false)) {
             getWindow().addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);
         }
+
         // Fetch openHAB service type name from strings.xml
         openHABServiceType = getString(R.string.openhab_service_type);
+
         // Get username/password from preferences
         openHABUsername = mSettings.getString(Constants.PREFERENCE_USERNAME, null);
         openHABPassword = mSettings.getString(Constants.PREFERENCE_PASSWORD, null);
         mAsyncHttpClient.setBasicAuth(openHABUsername, openHABPassword, true);
         mAsyncHttpClient.setTimeout(30000);
-        if (!isDeveloper)
+
+        if (!BuildConfig.IS_DEVELOPER) {
             Util.initCrittercism(getApplicationContext(), "5117659f59e1bd4ba9000004");
+        }
+
         Util.setActivityTheme(this);
         super.onCreate(savedInstanceState);
-        if (!isDeveloper)
+
+        if (!BuildConfig.IS_DEVELOPER) {
```

Comment:
removed "isDeveloper" local boolean and put it in the build.gradle as a BuildConfig property. This can now be used app wide and is less prone to being left true on production builds